README.md

```diff
@@ -1 +1,51 @@
-# bitrise-step-restore-xcodebuild-cache
\ No newline at end of file
+# Restore xcodebuild Bitrise Build Cache
+
+[![Step changelog](https://shields.io/github.com/bitrise-steplib/bitrise-step-restore-xcodebuild-cache?include_prereleases&label=changelog&color=blueviolet)](https://github.com/bitrise-steplib/bitrise-step-restore-xcodebuild-cache/releases)
+
+Restores the DerivedData folder and the related metadata from Bitrise Build Cache to speed up subsequent builds.
+
+<details>
+<summary>Description</summary>
+
+This step is the pair of the [save-xcodebuild-cache step](https://github.com/bitrise-steplib/bitrise-step-save-xcodebuild-cache) and can be used to restore xcodebuild cache archives from the Bitrise Build Cache. 
+
+As xcodebuild only reuses DerivedData if the input files' modification time is the same, the step also restores the modification time of the input files (project files including source code files) from a metadata file stored along the
```

Comment:
> use the same build settings/settingsas

What did you want to write here?

Reply:
Please edit step.yml for description, as README is autogenerted.

Reply:
oh, good to know :D